glauth/docs/getting-started.md

```diff
@@ -0,0 +1,272 @@
+
+## Usage
+
+The program provides a few sub-commands on execution. The available configuration methods have already been mentioned above. Generally you can always see a formated help output if you execute the binary via `ocis-glauth --help`.
+
+### Server
+
+The server command is used to start the http and debug server on two addresses within a single process. The http server is serving the general webservice while the debug server is used for health check, readiness check and to server the metrics mentioned below. For further help please execute:
+
+{{< highlight txt >}}
+ocis-glauth server --help
+{{< / highlight >}}
+
+### Health
+
+The health command is used to execute a health check, if the exit code equals zero the service should be up and running, if the exist code is greater than zero the service is not in a healthy state. Generally this command is used within our Docker containers, it could also be used within Kubernetes.
+
+{{< highlight txt >}}
+ocis-glauth health --help
+{{< / highlight >}}
+
+## Metrics
+
+This service provides some [Prometheus](https://prometheus.io/) metrics through the debug endpoint, you can optionally secure the metrics endpoint by some random token, which got to be configured through one of the flag `--debug-token` or the environment variable `GLAUTH_DEBUG_TOKEN` mentioned above. By default the metrics endpoint is bound to `http://0.0.0.0:9124/metrics`.
+
+go_gc_duration_seconds
+: A summary of the GC invocation durations
+
+go_gc_duration_seconds_sum
+: A summary of the GC invocation durations
+
+go_gc_duration_seconds_count
+: A summary of the GC invocation durations
+
+go_goroutines
+: Number of goroutines that currently exist
+
+go_info
+: Information about the Go environment
+
+go_memstats_alloc_bytes
+: Number of bytes allocated and still in use
+
+go_memstats_alloc_bytes_total
+: Total number of bytes allocated, even if freed
+
+go_memstats_buck_hash_sys_bytes
+: Number of bytes used by the profiling bucket hash table
+
+go_memstats_frees_total
+: Total number of frees
+
+go_memstats_gc_cpu_fraction
+: The fraction of this program's available CPU time used by the GC since the program started
+
+go_memstats_gc_sys_bytes
+: Number of bytes used for garbage collection system metadata
+
+go_memstats_heap_alloc_bytes
+: Number of heap bytes allocated and still in use
+
+go_memstats_heap_idle_bytes
+: Number of heap bytes waiting to be used
+
+go_memstats_heap_inuse_bytes
+: Number of heap bytes that are in use
+
+go_memstats_heap_objects
+: Number of allocated objects
+
+go_memstats_heap_released_bytes
+: Number of heap bytes released to OS
+
+go_memstats_heap_sys_bytes
+: Number of heap bytes obtained from system
+
+go_memstats_last_gc_time_seconds
+: Number of seconds since 1970 of last garbage collection
+
+go_memstats_lookups_total
+: Total number of pointer lookups
+
+go_memstats_mallocs_total
+: Total number of mallocs
+
+go_memstats_mcache_inuse_bytes
+: Number of bytes in use by mcache structures
+
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (145:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)